src/modules/quoteSummary-iface.ts

```diff
@@ -1431,6 +1431,12 @@ export const QuoteSummaryResult = Type.Object(
     summaryProfile: Type.Optional(SummaryProfile),
     topHoldings: Type.Optional(TopHoldings),
     upgradeDowngradeHistory: Type.Optional(UpgradeDowngradeHistory),
+    indexTrend: Type.Optional(IndexTrend),
```

Comment:
These I actually forgot on my last PR and was picked up by `_internalDisallowAdditionalProperties` flag changes

Reply:
Haha great to see that one already paying for itself ;)